src/app/search/search-filters.ts

```diff
@@ -847,17 +848,17 @@ export function searchFilters(
           "BUCKET_SPECIAL_WEAPON",
           "BUCKET_HEAD",
           "BUCKET_ARMS",
-          "BUCKET_GHOST",
-          3448274439,
-          3551918588,
-          14239492,
-          20886954,
-          1585787867,
-          1498876634,
-          2465295065,
-          953998645
+          "BUCKET_GHOST"
         ];
-        return item.primStat && item.bucket && _.contains(lightBuckets, item.bucket.id);
+        const lightBucketsSort = [
+          "Armor",
+          "Weapons"
+        ];
```

Comment:
Hmm, this'll include subclasses won't it?

In the max light loadout, we use a check like this:

```ts
const statHashes = new Set([
    1480404414, // D2 Attack
    3897883278, // D1 & D2 Defense
    368428387 // D1 Attack
  ]);

  const applicableItems = storeService.getAllItems().filter((i) => {
    return i.canBeEquippedBy(store) &&
      i.primStat && // has a primary stat (sanity check)
      statHashes.has(i.primStat.statHash); // one of our selected stats
  });
```

Reply:
Nope:
![screenshot_20180711_191406](https://user-images.githubusercontent.com/4798491/42607169-c8fb11b8-853e-11e8-94fa-315ba7fe6c1d.png)

Reply:
Can you try the check I posted? It should work in both D1 and D2 with less hardcoding.

Reply:
@bhollis fixed